client/src/annoMatrix/views.ts

```diff
@@ -188,10 +237,17 @@ export class AnnoMatrixRowSubsetView extends AnnoMatrixView {
 Utility functions below
 */
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'field' implicitly has an 'any' type.
-function _clipAnnoMatrix(field, colLabel, colSchema, colData, df, qmin, qmax) {
+function _clipAnnoMatrix(
+  field: Field,
+  colLabel: string,
+  colSchema: ColumnSchema,
+  colData: string[],
+  df: Dataframe,
+  qmin: number,
+  qmax: number
+): string[] {
   /* only clip obs and var scalar columns */
-  if (field !== "obs" && field !== "X") return colData;
+  if (field !== Field.obs && field !== Field.X) return colData;
   if (!_isContinuousType(colSchema)) return colData;
   if (qmin < 0) qmin = 0;
   if (qmax > 1) qmax = 1;
```

Comment:
suggest that typing be added to `clip()` (below) as part of this PR.     util/clip.ts

Reply:
Updated in #2381.